Makefile

```diff
@@ -50,10 +50,10 @@ uninstall:
 	helm uninstall $(NAME)
 
 vault-init:
-	common/scripts/vault-utils.sh vault_init common/pattern-vault.init
+	common/scripts/vault-utils.sh vault_init $(shell readlink -f common/pattern-vault.init)
 
 vault-unseal:
-	common/scripts/vault-utils.sh vault_unseal common/pattern-vault.init
+	common/scripts/vault-utils.sh vault_unseal $(shell readlink -f common/pattern-vault.init)
```

Comment:
redundant readlink call?

(got to pretend to be useful somehow)

Reply:
:) I kinda liked having absolute filenames there and I think there was also another reason for me to move to them, but I tested without and it seems to work from outside common and from inside common (thanks to the the common -> . symlink I suppose). But am not married to it, will remove it ;)